doc/release_notes_0.8.rst

```diff
@@ -0,0 +1,392 @@
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation.  You can either download this source code as a zip or
+tar.gz file (see the `ACRN v0.8 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.8>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.8
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.8 documents can be found at
+https://projectacrn.github.io/0.8/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+ACRN v0.8 requires Clear Linux OS version 28600 or newer.  Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.8 new features
+************************
+
+GPIO virtualization
+=========================
+
+GPIO virtualization is supported as para-virtualization based on the
+Virtual I/O Device (VIRTIO) specification. The GPIO consumers of the
+Front-end are able to set or get GPIO values, directions, and
+configuration via one virtual GPIO controller. In the Back-end, the GPIO
+command line in the launch script can be modified to map native GPIO to
+UOS.
+
+Enable QoS based on runC container
+==================================
+ACRN supports Device-Model QoS based on runC container to control the SOS
+resources (CPU, Storage, MEM, NET) by modifying the runC configuration file.
+
+S5 support for RTVM
+===============================
+ACRN supports a Real-time VM (RTVM) shutting itself down. A RTVM is a
+kind of VM that the SOS can't interfere at runtime, and as such, can
+only power itself off internally. All poweroff requests external to the
+RTVM will be rejected to avoid any interference.
+
+Document updates
+================
+Several new documents have been added in this release, including:
+
+* :ref:`Zephyr RTOS as Guest OS <using_zephyr_as_uos>`
+* :ref:`Enable cache QoS with CAT <using_cat_up2>`
+* :ref:`ACRN kernel parameter introduction <kernel-parameters>`
+* :ref:`faq` update for two issues
+* :ref:`ACRN Debug introduction <acrn-debug>`
+
+New Features Details
+********************
+
+- :acrn-issue:`923` - GPU Mediator shall be compatible with operation of graphics safety watchdog
+- :acrn-issue:`1409` - Add support for profiling [sep/socwatch tools]
+- :acrn-issue:`1568` - Implement PCI emulation functionality in HV for UOS passthrough devices and SOS MSI/MSI-X remapping
+- :acrn-issue:`1867` - vMSR code reshuffle
+- :acrn-issue:`2020` - DM: Enable QoS in ACRN, based on runC container
+- :acrn-issue:`2512` - GPIO virtualization
+- :acrn-issue:`2605` - hv: search additional argument when parsing seed from ABL(apl_sdc_stable)
```

Comment:
@NanlinXie recommends deleting this issue from the list

Reply:
OK